tests/unit/utils_test.py

```diff
@@ -299,56 +299,30 @@ def test_convert_volume_binds_no_mode(self):
         self.assertEqual(convert_volume_binds(data), ['/mnt/vol1:/data:rw'])
 
     def test_convert_volume_binds_unicode_bytes_input(self):
-        if six.PY2:
```

Comment:
I don't think we want to remove the tests for PY2. Having them in separate test cases with `pytest.mark.skip` would be fine.